mdata/store_cassandra.go

```diff
@@ -435,12 +449,20 @@ func (c *CassandraStore) SearchTable(key, table string, start, end uint32) ([]ch
 	results := make(chan outcome, numQueries)
 	for i := range crrs {
 		crrs[i].out = results
-		c.readQueue <- crrs[i]
+		select {
+		case c.readQueue <- crrs[i]:
+		default:
+			cassReadQueueFull.Inc()
+			return nil, errReadQueueFull
+		}
```

Comment:
I don't see why the behavior sholud change from blocking to drop-on-full.
I see there was some discussion https://github.com/raintank/metrictank/pull/685#pullrequestreview-49001631 but I don't get it. the code is pretty clear that the timestamps are set when CRR are created, before they are put in to the queue, and we look at the age after pulling from queue. so the age includes time waited to put into queue